src/main/java/com/lina/airline/controller/UserController.java

```diff
@@ -18,15 +18,23 @@
 import java.util.UUID;
 
 @RestController
-@RequestMapping("/users")
+//@RequestMapping("/api/v1'")
 @RequiredArgsConstructor
 public class UserController {
 
    public final UserService userService;
 
-    @PostMapping("/")
-    public UUID registrationUser(@RequestBody @Valid UserRegistrationRequest userRegistrationRequest){
-        return userService.saveUser(userRegistrationRequest);
+    @PostMapping("/register")
+    public ResponseEntity<APIResponse<CreationResponse>> registrationUser(@RequestBody @Valid UserRegistrationRequest userRegistrationRequest){
```

Comment:
This is a completely different task. Never do multiple task in same commit. It is OK for now